tests/z_api_alignment_test.c

```diff
@@ -215,9 +221,9 @@ int main(int argc, char **argv) {
     assert(hellos >= 1);
 
     uint32_t _scouting_timeout = strtoul(SCOUTING_TIMEOUT, NULL, 10);
-    z_sleep_ms(_scouting_timeout);
+    sleep(_scouting_timeout / 1000);
```

Comment:
In Zenoh-Pico/Zenoh-C we should better use `z_sleep_*` and avoid using platform-specific syscalls.
However, those functions are still missing in Zenoh-C (see https://github.com/eclipse-zenoh/zenoh-c/issues/183).
Nevertheless, they are present in Zenoh-Pico, so better keep them.

Reply:
But as far as I understand, the idea is for the files to be identical in both repos, do you think adding a `#define` here would be reasonable?

Reply:
I would say that the additional functions shall be added in zenoh-c, as referred by @Mallets.

Reply:
Yes, sure, but until we have no them in zenohc, api alignment test on them has no sense.